apps/meteor/server/lib/resetUserE2EKey.ts

```diff
@@ -64,6 +65,9 @@ export async function resetUserE2EEncriptionKey(uid: string, notifyUser: boolean
 		throw new Meteor.Error('error-not-allowed', 'Federated Users cant have TOTP', { function: 'resetTOTP' });
 	}
 
+	// force logout the live sessions
+	Notifications.notifyUser(uid, 'force_logout');
```

Comment:
```suggestion
	Notifications.notifyUser(uid, 'force-logout');
```
the pattern used in other streams is dash and not underlining.

https://github.com/RocketChat/Rocket.Chat/blob/a7823c1b0901c510af5bfa994e7b3f96ee10dd91/ee/packages/ddp-client/src/legacy/types/SDKLegacy.ts#L77

Reply:
There's already a stream set with this name.
```
{ key: `${string}/force_logout`; args: [] },
```
it's used here: apps/meteor/client/startup/forceLogout.ts

I think this was introduced together with the Device Management feature. Not sure we should change it now...

Reply:
Oops, I missed your comment Daniel.

Gabriel is right, this is not a new stream event, we can change the stream event but not sure, maybe we should discuss first with architecture team. 🤔

Reply:
Ok, I understand that it was done like this in the past, but the value that will come from the stream will look like this:
`stream-force_logout`.
Strange isn't it?

As this is already closed for 6.8, I don't see any problems, but I think it's worth coming back to this later and following the previous pattern.